storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -160,7 +160,10 @@ export const CheckboxGroup: Story = {
     legend: 'Waar gaat uw melding over?',
   },
   render: (args) => (
-    <FieldSet legend={args.legend} invalid={args.invalid}>
+    <FieldSet aria-labelledby="fieldset1 description6" id="fieldset1" invalid={args.invalid} legend={args.legend}>
```

Comment:
Does the content of `#description6` really add value to the Field Set label on top of its own legend property? A description seems additive to me and it is already visible right below.

Reply:
> it is already visible right below

It isn't for screen readers. When you tab to an input in a form, a screen reader only reads whatever is connected to that input. If you don't connect the description, it isn't read. Normally, you would use `aria-describedby` for this, but that doesn't work with NVDA for Field Sets containing Checkboxes because of the bug linked to in the docs.